kubetest/gke.go

```diff
@@ -55,6 +55,8 @@ var (
 	gkeReleaseChannel              = flag.String("gke-release-channel", "", "(gke only) if specified, bring up GKE clusters from that release channel.")
 	gkeSingleZoneNodeInstanceGroup = flag.Bool("gke-single-zone-node-instance-group", true, "(gke only) Add instance groups from a single zone to the NODE_INSTANCE_GROUP env variable.")
 	gkeNodePorts                   = flag.String("gke-node-ports", "", "(gke only) List of ports on nodes to open, allowing e.g. master to connect to pods on private nodes. The format should be 'protocol[:port[-port]],[...]' as in gcloud compute firewall-rules create --allow.")
+	gkeNat                         = flag.Bool("gke-nat", false, "(gke only) Configure Cloud NAT allowing outbound connections in cluster with private nodes.")
```

Comment:
Given that this is a bool flag, can we rename it to be something like `gke-set-up-nat` or `gke-create-nat`?

Reply:
OK.
create-nat sounds better